notebooks/pandas_07_reshaping_data.md

```diff
@@ -243,204 +422,27 @@ df.pivot_table(index='Underaged', columns='Sex',
                values='Fare', aggfunc='median')
 ```
 
-# Melt - from pivot table to long or tidy format
-
-+++
-
-The `melt` function performs the inverse operation of a `pivot`.
-
-```{code-cell} ipython3
-pivoted = df.pivot_table(index='Sex', columns='Pclass', values='Fare').reset_index()
-pivoted.columns.name = None
-```
-
-```{code-cell} ipython3
-pivoted
-```
-
-Assume we have a DataFrame like the above. The observations (the average Fare people payed) are spread over different columns. To make sure each value is in its own row, we can use the `melt` function:
-
-```{code-cell} ipython3
-pd.melt(pivoted)
-```
-
-As you can see above, the `melt` function puts all column labels in one column, and all values in a second column.
-
-In this case, this is not fully what we want. We would like to keep the 'Sex' column separately:
-
-```{code-cell} ipython3
-pd.melt(pivoted, id_vars=['Sex']) #, var_name='Pclass', value_name='Fare')
```

Comment:
And also removed this old melt example using a pivoted subset of the titanic data, as we now use the WWTP data to illustrate it